src/components/button-toggle/button-toggle.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  Component,
+  ContentChildren,
+  Directive,
+  EventEmitter,
+  HostBinding,
+  HostListener,
+  Input,
+  OnInit,
+  Optional,
+  Output,
+  QueryList,
+  ViewEncapsulation,
+  forwardRef
+} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {
+  MdUniqueSelectionDispatcher
+} from '@angular2-material/core/coordination/unique-selection-dispatcher';
+
+
+
+var _uniqueIdCounter = 0;
+
+/** A simple change event emitted by either MdButtonToggle or MdButtonToggleGroup. */
+export class MdButtonToggleChange {
+  source: MdButtonToggle;
+  value: any;
+}
+
+/** Exclusive selection button toggle group. */
+@Directive({
+  selector: 'md-button-toggle-group:not([multiple])',
+  host: {
+    'role': 'radiogroup',
+  },
+})
+export class MdButtonToggleGroup {
+  /** The value for the button toggle group. Should match currently selected button toggle. */
+  private _value: any = null;
+
+  /** The HTML name attribute applied to toggles in this group. */
+  private _name: string = `md-radio-group-${_uniqueIdCounter++}`;
+
+  /** Disables all toggles in the group. */
+  private _disabled: boolean = null;
+
+  /** The currently selected button toggle, should match the value. */
+  private _selected: MdButtonToggle = null;
+
+  /** Event emitted when the group's value changes. */
+  private _change: EventEmitter<MdButtonToggleChange> = new EventEmitter<MdButtonToggleChange>();
+  @Output() get change(): Observable<MdButtonToggleChange> {
+    return this._change.asObservable();
+  }
+
+  /** Child button toggle buttons. */
+  @ContentChildren(forwardRef(() => MdButtonToggle))
+  private _buttonToggles: QueryList<MdButtonToggle> = null;
```

Comment:
This one is used, I removed the unused one in the MdButtonToggleGroupMultiple though.